src/main/resources/com/microsoft/jenkins/containeragents/aci/networkProfileSnippet.json

```diff
@@ -0,0 +1,24 @@
+{ "type": "Microsoft.Network/networkProfiles",
```

Comment:
This seems to be deprecated on the latest API version possibly we should upgrade and only use the new version?
https://docs.microsoft.com/en-us/azure/container-instances/container-instances-virtual-network-concepts#network-profile

Reply:
I oriented myself on Azure examples. I think the whole deployment template need an update. Maybe a separate issue, where the full deployment template can be updated.